### PR TITLE
Copy editing (especially in ImageMosaic sections)

### PR DESCRIPTION
**Pull request by maddenp**
I have been carefully reading the ImageMosaic sections to address some issues I am having, and have noticed some typos, inconsistencies and syntax issues I thought I would try to help correct.

In many files, the only changes are to consistently capitalize "GeoServer" and "ImageMosaic". There are probably still instances of "geoserver" used where "GeoServer" should be, but these are more difficult to differentiate from legitimate uses in filenames and code examples, and I don't have time at the moment to hunt them all down.

In other places, I have made changes that I believe lead to more idiomatic English.

I have made inline comments in other places where I thought my intent might be less obvious.

### CODE REVIEW

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst [hunk 1]
```diff
@@ -73,11 +73,11 @@ The table below describes the various elements in this configuration file.
      - Y
      - Suggested plugin for reading the image files.
    * - Envelope2D
-     - Y
+     - N
```
Comment (maddenp) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: This option seems not to actually be required. In any case, it does not appear in the example configuration shown below, so either this line or the configuration is wrong. (That is, shouldn't the option appear in the example configuration if it is mandatory?)

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst [hunk 2]
```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
      - N
      - Path to an auxiliary file to be used for internal purposes (For example: when dealing with NetCDF granules, it refers to the NetCDF XML ancillary file.)
    * - AbsolutePath
-     - Y
-     - Controls whether or not the path stored inside the ``location`` attribute represents an absolute path or a path relative to the location of the shapefile index. Notice that a relative index ensures much more portability of the mosaic itself. Default value for this parameter is ``false``, which means relative paths.
+     - N
```
Comment (maddenp) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: Similar to the previous comment, either this option is not actually required, or the example configuration shown below should include it (I think).

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst [hunk 3]
```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
...
    * - Caching
      - N
-     - Boolean value to enable caching. When set to ``true``, the imagemosaic will try to save in memory the entire contents of the index to reduce loading/query time. Set to ``false`` for a large granule index and/or if new granules are to be ingested (for example, when the index is on a database and we interact directly with it). Default is ``false``.
+     - Boolean value to enable caching. When set to ``true``, the ImageMosaic will try to save in memory the entire contents of the index to reduce loading/query time. Set to ``false`` for a large granule index and/or if new granules are to be ingested (for example, when the index is on a database and we interact directly with it). Default is ``false``.
    * - CanBeEmpty
      - N
-     - Boolean flag used for configuring empty mosaics. When enabled the imagemosaic will not throw an exception caused by the absence of any coverage. By default it is set to false.
+     - Boolean flag used for configuring empty mosaics. When enabled the ImageMosaic will not throw an exception caused by the absence of any coverage. By default it is set to ``false``.
```
Comment (maddenp) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: In a number of places, I have added double-backticks around boolean values for consistency. Mostly, the original document had them in backticks, so I tried treat all instances the same.

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst [hunk 4]
```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
...
    * - Envelope2D
-     - Y
+     - N
```
Comment (maddenp) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: Again, apparently not required, or missing from the given example config.

File: doc/en/user/source/data/raster/imagemosaic/tutorial.rst [hunk 5]
```diff
@@ -318,13 +318,13 @@ Multi-resolution imagery with reprojection
 
 As a general rule, we want to have the highest resolution granules shown "on top", with the lower-resolution granules filling in the gaps as necessary.
 
-In this example, we will serve up overlapping granules that have varying resolutions. In addition, we will mix resolutions, such that the higher resolution granule is reprojected to match the resolution of the 
+In this example, we will serve up overlapping granules that have varying resolutions. In addition, we will mix resolutions, such that the higher resolution granule is reprojected to match the resolution of the lower resolution granules.
```
Comment (maddenp) [inline doc/en/user/source/data/raster/imagemosaic/tutorial.rst]: This sentence was previously incomplete, so I have guessed here about the correct end. I hope I got it right.

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst [hunk 6]
```diff
@@ -74,7 +74,7 @@ The table below describes the various elements in this configuration file.
      - Suggested plugin for reading the image files.
    * - Envelope2D
      - N
-     - Envelope for the mosaic formatted as ```LLX,LLY URX,URY`` (notice the space between the lower left and upper right coordinate pairs).
+     - Envelope for the mosaic formatted as ``LLX,LLY URX,URY`` (notice the space between the lower left and upper right coordinate pairs).
```
Comment (maddenp) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: I *think* the exta backtick is not actually part of the syntax as it should appear in the configuration file.

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst [hunk 7]
```diff
@@ -104,11 +104,11 @@ A sample configuration file follows::
 
 By default the ImageMosaic index is specified by a shapefile, which is located at the root of the ImageMosaic directory, just like the primary configuration file.
 
-If needed different storage can be used for the index like a spatial DBMS which is the preferred solution when willing to share the ImageMosaic itself in a cluster of GeoServer intances. In this case the user must supply GeoSerer with the proper onnection parameters which can be specified by using a :file:`datastore.properties` file placed at the root of the ImageMosaic directory.
+If needed, different storage can be used for the index -- like a spatial DBMS, which is the preferred solution when you wish to share the ImageMosaic itself in a cluster of GeoServer instances. In this case the user must supply GeoServer with the proper connection parameters, which can be specified by using a :file:`datastore.properties` file placed at the root of the ImageMosaic directory.
```
Comment (tbarsballe) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: I don't think there should be two dashes (`--`) here. Were you trying to use an em-dash?

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst (hunk 2 above)
Comment (dvntucker) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: AbsolutePath is *not* required.

File: doc/en/user/source/data/raster/imagemosaic/tutorial.rst [hunk 8]
```diff
@@ -239,8 +239,8 @@ The result is the following.
 
    Basic configuration
 
-This mosaic, formed by two single granules, shows a typical case where the "nodata" collar areas of the granules overlap, as hown in the picture above.
-In this case we can use the ``InputTransparentColor`` parameter to make the collar areas disappear during the superimposition process, as instance, in this case, by using an ``InputTransparentColor`` of ``#FFFFFF``  
+This mosaic, formed by two single granules, shows a typical case where the "nodata" collar areas of the granules overlap, as shown in the picture above.
+In this case we can use the ``InputTransparentColor`` parameter to make the collar areas disappear during the superimposition process -- in this case, by using an ``InputTransparentColor`` of ``#FFFFFF``.
```
Comment (tbarsballe) [inline doc/en/user/source/data/raster/imagemosaic/tutorial.rst]: Should this be an em-dash (—)?

File: doc/en/user/source/data/raster/imagemosaic/tutorial.rst (hunk 5 above)
Comment (tbarsballe) [inline doc/en/user/source/data/raster/imagemosaic/tutorial.rst]: That seems like the correct ending.

File: doc/en/user/source/data/raster/imagemosaic/configuration.rst (hunk 7 above)
Comment (maddenp) [inline doc/en/user/source/data/raster/imagemosaic/configuration.rst]: I'll fix this throughout...

**Comment by maddenp** [comment]
Hi Torben -- I squashed the commits, hopefully correctly. Please have a look and see if it's ok. I also changed the double-hyphens to em-dashes as suggested.

**Comment by tbarsballe** [comment]
Looks good, thanks for all the fixes. Merging.

**Pull request merged**